build/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM debian:bullseye@sha256:71cb300d5448af821aedfe63afd55ba05f45a6a79f00dcd131b96b780bb99fe4
+
+ENV CORE_RS_ALBATROSS_COMMIT_HASH=93066288a6bfccc43498ae5596cdf2189b244785
+ENV CORE_RS_ALBATROSS_DIGEST=f606933de4341821be1ab765167c2637c20afb0b50b889b4311242f4a407259d
+
+# Set system time to 00:00:01 UTC
+ENV SOURCE_DATE_EPOCH=1
+
+# Avoid prompts during installation
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Copy apt install files from repo
+COPY apt/packages/ ./
+
+# Copy digest for apt install files
+COPY apt/apt.lockfile ./
+
+# Install all deb files
+RUN dpkg -i *.deb || true
+
+# Create nimiq directory for build
+RUN mkdir -p /home/nimiq/
+
+# Change homedir
+WORKDIR /home/nimiq
+
+# Clone core-rs-albatross repo
+RUN curl -LO https://github.com/nimiq/core-rs-albatross/archive/${CORE_RS_ALBATROSS_COMMIT_HASH}.zip \
```

Comment:
Do we need to re-clone the repo? If people get here they probably already cloned the repo

Reply:
To have the deterministic build hashes match, we need to make sure that whatever this dockerfile is building matches the latest release made by Nimiq. 

If the latest commit/tag always corresponds to the latest release we could make the adjustment you are suggesting, otherwise we need to explicitly set the commit/tag that's meant to be used for the latest release.

Reply:
The code has been updated to use the latest tag for the build.